src/components/autocompleteTagsComponent.js

```diff
@@ -93,35 +110,40 @@ const autocompleteTagsComponent = forwardRef((props, ref) => {
     props.onChange(createPatchFrom(!value ? [] : value))
   }
 
-  /* 
+  /*
   Ok, here's some fun: here we handle changes that involve creating new tags and populating these new options into selected tags and all tags
   */
-  const createOption = inputValue => {
-    let newSelected = selected
-    newSelected.push({ value: inputValue, label: inputValue })
-    setSelected(newSelected)
-
-    // New tags need to be commited to Sanity so that we can reuse them elsewhere
-    client
-      .patch(props.document._id)
-      .append(document, [{ value: inputValue, label: inputValue }])
-      .commit()
-      .then(() => props.onChange(createPatchFrom(newSelected)))
+  const selectMenuProps = {
+    disabled: isLoading,
+    isLoading,
+    values: selected ? selected : [],
+    isMulti: true,
+    options: uniqueTags || "",
+    onChange: handleChange,
+    closeMenuOnSelect
+  }
+  if (!frozen) {
+    selectMenuProps.onCreateOption = inputValue => {
+      let newSelected = selected
+      newSelected.push({ value: inputValue, label: inputValue })
+      setSelected(newSelected)
+
+      // New tags need to be commited to Sanity so that we can reuse them elsewhere
+      client
+        .patch(props.document._id)
+        .append(document, [{ value: inputValue, label: inputValue }])
+        .commit()
+        .then(() => props.onChange(createPatchFrom(newSelected)))
+    }
   }
-
   return (
-    <>
-      <h4>{props.type.title}</h4>
-      <CreatableSelect
-        disabled={isLoading}
-        isLoading={isLoading}
-        value={selected ? selected : []}
-        isMulti
-        onChange={handleChange}
-        onCreateOption={createOption}
-        options={uniqueTags || ""}
-      />
-    </>
+    <Fieldset level={level} legend={type.title} description={type.description}>
```

Comment:
This provides proper nesting if tag is in a fieldset:
![Screen Shot 2021-02-01 at 4 51 46 PM](https://user-images.githubusercontent.com/177857/106536740-e17b3500-64ad-11eb-8c28-f5245e660006.png)